### ISSUE TITLE
doesn't compiled in Xcode 8 

### ISSUE THREAD

**Issue by skv-headless** [tradle/react-native-udp#26]
First error in websocket build. here fix https://github.com/facebook/react-native/issues/8584#issuecomment-236366222
Second 
```
XCode has selected SDK: iphonesimulator with version: 0.0 (although back-targetting: 7.0)
...therefore, OTHER_SDK_TO_BUILD = iphoneos0.0
RECURSION: I am the root ... recursing all missing build targets NOW...
RECURSION: ...about to invoke: xcodebuild -configuration "Debug" -project "UdpSockets.xcodeproj" -target "UdpSockets" -sdk "iphoneos0.0" build RUN_CLANG_STATIC_ANALYZER=NO BUILD_DIR="/Users/headless/Library/Developer/Xcode/DerivedData/rctsockets-dbipadwhaxgiauewyahdiinokyed/Build/Products" BUILD_ROOT="/Users/headless/Library/Developer/Xcode/DerivedData/rctsockets-dbipadwhaxgiauewyahdiinokyed/Build/Products" SYMROOT="/Users/headless/Library/Developer/Xcode/DerivedData/rctsockets-dbipadwhaxgiauewyahdiinokyed/Build/Products"
xcodebuild: error: SDK "iphoneos0.0" cannot be located.
Command /bin/sh failed with exit code 64
```
Dont know how to fix yet

**Comment by joefocusrite** [comment]
It think this is fixed in #25...

**Comment by skv-headless** [comment]
Yep. You are write.

**Comment by mvayngrib** [comment]
merged #25 and released in 1.2.6

**Comment by skv-headless** [comment]
Thanks 👍
